packages/flx/test/signing_test.dart

```diff
@@ -34,7 +34,7 @@ main() async {
 
   // Set up a key generator.
   CipherParameters cipher = CipherParameters.get();
-  await cipher.seedRandom();
+  cipher.seedRandom();
```

Comment:
Yeah, it was just async to read a few bytes from a file, and that can be done just as easily w/ sync read. I generally prefer `fooSync()` to `await fooAsync()` when a sync version of a method is available.